File: app/src/main/java/com/di/penopllast/vklikesremover/entity/RootFaveGetPost.java

package com.di.penopllast.vklikesremover.entity;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class RootFaveGetPost {

    @SerializedName("response")
    @Expose
    private ResponseFaveGetPosts response;

    public ResponseFaveGetPosts getResponse() {
        return response;
    }

    public void setResponse(ResponseFaveGetPosts response) {
        this.response = response;
    }

}

File: app/src/main/java/com/di/penopllast/vklikesremover/entity/like/RootFaveGetPost.kt
package com.di.penopllast.vklikesremover.entity.like

import com.di.penopllast.vklikesremover.entity.RootRetrofitResponse
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class RootFaveGetPost : RootRetrofitResponse {

    @SerializedName("response")
    @Expose
    val response: ResponseFaveGetPosts? = null

}
